app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/IntentHelper.java

```diff
@@ -0,0 +1,28 @@
+package io.github.droidkaigi.confsched2017.view.helper;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.support.annotation.NonNull;
+
+import timber.log.Timber;
+
+public class IntentHelper {
+
+    /**
+     * Builds an intent of type Intent.ACTION_VIEW from the passed htmlUrl.
+     * But, When intent.resolveActivity(context.getPackageManager()) == null is true, it return null.
+     */
+    public static Intent buildActionViewIntent(@NonNull Context context, @NonNull String htmlUrl) {
```

Comment:
@konifar 
Optional requires API 24 or more.
So I add `@Nullable`.

Is there any other good way?